ChangeSet
  1.1526.1.1 04/03/29 17:04:29 dgibson@wocket.ozlabs.ibm.com +1 -0
  Fix iSeries virtual ethernet driver for pci_dma_error -> pci_dma_mapping_error
  change.

  drivers/net/iseries_veth.c
    1.12 04/03/29 17:03:57 dgibson@wocket.ozlabs.ibm.com +1 -1
    Fix iSeries virtual ethernet driver for pci_dma_error -> pci_dma_mapping_error
    change.

diff -Nru a/drivers/net/iseries_veth.c b/drivers/net/iseries_veth.c
--- a/drivers/net/iseries_veth.c	Tue Mar 30 00:07:32 2004
+++ b/drivers/net/iseries_veth.c	Tue Mar 30 00:07:32 2004
@@ -455,7 +455,7 @@
 	
 	/* Is it really necessary to check the length and address
 	 * fields of the first entry here? */
-	if (!pci_dma_error(dma_address)) {
+	if (!pci_dma_mapping_error(dma_address)) {
 		msg->skb = skb;
 		msg->data.addr[0] = dma_address;
 		msg->data.len[0] = dma_length;
.........................................................................
# vim: syntax=diff

